css/style.css

```diff
@@ -0,0 +1,94 @@
+html {
```

Comment:
Kindly configure the linter for this project as it will help you to avoid linter errors and will save your time and will increase your productivity.😄 😉😄 😉